test/backend/inference_backend.py

```diff
@@ -283,6 +283,11 @@ def get_test_models():
         # ==OP== DynamicQuantizeLinear
 
         # ==OP== EinSum
+        "test_einsum_batch_diagonal_cpu": {STATIC_SHAPE:{}, DYNAMIC_SHAPE:{-1:{-1}}, CONSTANT_INPUT:{-1}},
```

Comment:
Could you please add a magic line (starting with `# ==LIM==`) about type limitation at L286, e.g.
```
# ==LIM== Limited to the types supported by ReduceSum and MatMul (which we decompose to in most cases) which exclude integers with width < 32
```
This magic line will be used to generate this document: https://github.com/onnx/onnx-mlir/blob/main/docs/SupportedONNXOps-cpu.md. You will find a python command to generate the document at the beginning of `SupportedONNXOps-cpu.md`. Please run the python command to re-generate the document.